feat: marketplace -> testing distribution and ent app store docs (#497)
appcircleio/appcircle-docusaurus: docs/marketplace/index.md

---
title: Marketplace Overview
description: Overview of Marketplace Integrations
tags: [testing-distribution, overview, concepts, marketplace]
sidebar_position: 1
---

> Appcircle provides variety extensions to establish integrations with other platforms

Reviewer: ```suggestion
Appcircle offers a variety of extensions to facilitate integrations with other platforms.
```
